sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
The added value of using https for SOURCE_URI is _just_ that it _looks_ consistent with using https for HOMEPAGE.
The added value of using the mirror redictor for SOURCE_URI is that the archives load faster and nobody will ever need to fetch it manually from a mirror if the main site becomes unavailable for e.g. a maintainance downtime.
What brings security is CHECKSUM_SHA256. Having https for SOURCE_URI does not bring anything more than _perceived consistency_.
So the choice is between _perceived consistency_ and _efficiency_.
The administrator of the GNU download site made the effort to build a mirror redirector service in the hope that it would be used. Not using it is just bad practice. Sorry if this reply looks rude.